### DISCUSSION TITLE
Run failed/cancelled: Build and Deploy - main

### DISCUSSION THREAD

**Discussion by CSpanias** [community/community#69178] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi all,

I am trying day and night to find out a way to fix the deployment issue, but it seems that I can't, thus, I thought to finally ask here! 

I created the [cspanias.github.io](https://github.com/CSpanias/cspanias.github.io) in order to host a basic blog I want. As you can see, I used the Jekyll Chirpy Starter template, and I firstly deployed the page without issues. Yesterday, I did a lot of changes/additions and I am receiving a ton of "**Run failed: Build and Deploy - main**" or "**Run cancelled: Build and Deploy - main**" messages. 

At first, I did not bother, as in the past the site usually updated after while, despite having received this message. 

Yesterday though, this was not the case. I was trying to solve the issue, without success, for 4-5 hours straight until I quitted. 

This morning, I can see some of the changes applied, but still a lot of them are missing. Could be a case that due to the volume of the changes, I created a huge "backlog" and it just needs its time to deploy the changes. Or time won't solve my issue and I am missing something else?

Thank you in advance for your time and help!

**Comment by atticusnari** [comment]
Bump. Having same issue and haven't figured it out yet either.
